shared/reducers/tracker.js

```diff
@@ -276,10 +303,15 @@ function updateUserState (state: TrackerState, action: Action): TrackerState {
       }
 
     case Constants.reportLastTrack:
-      const lastTrack = action.payload && action.payload.track
+      const currentlyFollowing = !!(action.payload && action.payload.track)
+      const proofs = state.proofs.map(p => ['btc', 'pgp'].includes(p.type)
```

Comment:
kinda crappy / would be fixed by sagas. the state of the key tracking is based on if you track them or not , so when that updates we need to also update the proofs